### PR TITLE
[enhancement] analyze filepath via glob

### PR DESCRIPTION
**Pull request by NumerHero**
realize a very very simple function via glob to analyze filepath~

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,11 +8,11 @@
   ],
   "main": "lib/index.js",
   "scripts": {
-    "clean": "del lib/ coverage/ example/build example/dist",
+    "clean": "rm -rf lib/ coverage/ example/build example/dist",
```
Comment (SimenB) [inline package.json]: Please revert this change

File: package.json [hunk 2]
```diff
@@ -43,17 +43,16 @@
     "babel-cli": "^6.18.0",
     "babel-plugin-add-module-exports": "^0.2.1",
     "babel-plugin-transform-async-to-module-method": "^6.16.0",
-    "babel-preset-env": "^1.4.0",
+    "babel-preset-es2015-node4": "^2.1.0",
```
Comment (SimenB) [inline package.json]: Changes from here and down should be reverted :smile:
Just rebase on master

File: src/handleUrl.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            : n.push(v)
+        );
+    } catch (e) {
+        throw new Error(e);
+    }
+
+    let b = [];
+    m.forEach(v => {
+        let t = v
+        let k = glob.sync(v.filepath);
```
Comment (SimenB) [inline src/handleUrl.js]: Use globby and `await` please

File: package.json (hunk 1 above)
Comment (SimenB) [inline package.json]: If you're a windows-user, master contains an update which should make this command work again

**Comment by SimenB** [comment]
@NumerHero Did you notice the feedback?

File: src/handleUrl.js (hunk 3 above)
Comment (NumerHero) [inline src/handleUrl.js]: sorry, I sure did't notice the feedback

**Comment by NumerHero** [comment]
I can't normaly run the example again. It's terribly strange. 

npm-error-log show that:

```
ReferenceError: Unknown plugin "transform-async-to-module-method" specified in "D:\\github\\program\\add-asset-html-webpack-plugin\\.babelrc" at 0, attempted to resolve relative to "D:\\github\\program\\add-asset-html-webpack-plugin"
```


I thought this very small and simple change.
so...How about update this change by yourselves ?

I want give up this pr. 

thx

**Comment by SimenB** [comment]
See #78
